pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -310,6 +310,8 @@ public void start() throws PulsarServerException {
 
     @Override
     public void disableBroker() throws Exception {
+        loadReportCacheZk.unregisterListener(this);
```

Comment:
I think we should add this logic to `stop()` method which is empty for both the loadManager right now.
And I think we should not delete the znode when we change the dynamic zk, but new load-manager should just update new load-repot on this path